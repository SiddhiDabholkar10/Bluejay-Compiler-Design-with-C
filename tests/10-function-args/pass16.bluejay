implicit i int := 473;

fun f int ()
  return i;


return f();
