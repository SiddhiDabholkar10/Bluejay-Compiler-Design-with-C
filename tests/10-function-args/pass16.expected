Function declared "f" returns int
	Implicit variable "i" type int position 0
Argument/local read "i" type int frame position 0
Function called "f" returns int
